packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1996,6 +1996,113 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(map1.get(testKey), testValue);
 		assert.strictEqual(map2.get(testKey), testValue);
 	});
+
+	describe("Negative tests for Offline Phase 3 - serializing without closing", () => {
+		it(`WRONGLY duplicates ops when submitted with different clientId from pendingLocalState (via Counter DDS)`, async function () {
+			const incrementValue = 3;
+			const pendingLocalState = await getPendingOps(
+				testContainerConfig,
+				provider,
+				true, // Do send ops from first container instance before closing
+				async (c, d) => {
+					const counter = await d.getSharedObject<SharedCounter>(counterId);
+					counter.increment(incrementValue);
+				},
+			);
+
+			// The real scenario where the clientId would differ from the original container and pendingLocalState is this:
+			// 1. container1 - getPendingLocalState (local ops have clientId A), reconnect, submitOp on new clientId B
+			// 2. container2 - load with pendingLocalState. There's no way to correlate the local ops from container1 (clientId A) with the remote ops from container1 (clientId B).
+			//
+			// For simplicity (as opposed to coding up reconnect like that), just tweak the clientId in pendingLocalState.
+			const obj = JSON.parse(pendingLocalState);
+			obj.clientId = "00000000-0e85-4ea1-983d-3cb72f280701"; // Bogus GUID to simulate reconnect after getPendingLocalState
+			const pendingLocalStateAdjusted = JSON.stringify(obj);
+
+			// load with pending ops with bogus clientId, which makes it impossible (today) to correlate the local ops and they're resent
+			const container2 = await loader.resolve({ url }, pendingLocalStateAdjusted);
+			const dataStore2 = (await container2.getEntryPoint()) as ITestFluidObject;
+			const counter2 = await dataStore2.getSharedObject<SharedCounter>(counterId);
+
+			await provider.ensureSynchronized();
+			assert.strictEqual(counter1.value, 2 * incrementValue);
```

Comment:
can you document what the correct values should be

Reply:
i think i would prefer if these test asserted the correct answer, but put a try catch around it, and ensure it breaks today. i think that would make it much clearer what should be happening in these cases, and how fixing it should appear

Reply:
I had thought of that but wanted to also assert the exact expected behavior, asserting what something is !== is not as strong as asserting what it is ===.  Maybe I'll do both

Reply:
I even imagined adding an `it.fails` thing that would make sure the test throws, but not worth it at this moment.